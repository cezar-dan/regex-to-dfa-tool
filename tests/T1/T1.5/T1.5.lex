 
SPACE
0
0,' ',1
1,' ',2
1

d
DS
0
0,'d',1
1,'d',1
1

ab
ABS
0
0,'a',1
0,'b',2
1,'a',2
1,'b',3
3,'a',1
3,'b',2
3 0

abc
ABCORC
0
0,'a',1
0,'b',2
0,'c',3
1,'a',2
1,'b',4
1,'c',2
3,'a',2
3,'b',2
3,'c',2
4,'a',2
4,'b',2
4,'c',5
5,'a',2
5,'b',2
5,'c',2
3 5

acd
APLUSCD
0
0,'a',1
0,'c',2
0,'d',2
1,'a',1
1,'c',3
1,'d',2
3,'a',2
3,'c',2
3,'d',4
4,'a',2
4,'c',2
4,'d',2
4

abd
ABD
0
0,'a',1
0,'b',2
0,'d',2
1,'a',2
1,'b',3
1,'d',2
3,'a',2
3,'b',2
3,'d',4
4,'a',2
4,'b',2
4,'d',2
4