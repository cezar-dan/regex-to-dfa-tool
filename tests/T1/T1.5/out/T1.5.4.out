ABD abd
SPACE  
ABCORC c
SPACE  
ABS abababababababababab
SPACE  